lib/active_model_serializers/model.rb

```diff
@@ -0,0 +1,43 @@
+# ActiveModelSerializers::Model is a convenient
+# serializable class to inherit from when making
+# serializable non-activerecord objects.
+module ActiveModelSerializers
+  class Model
+    include ActiveModel::Model
+    include ActiveModel::Serialization
```

Comment:
I don't think ActiveModel::Serialization is actually needed.
see: https://github.com/rails-api/active_model_serializers/pull/1241/files#diff-584dd3531b6814be034e8a6087f82fe9R22